pkg/inject/inject.go

```diff
@@ -211,6 +211,26 @@ func (g *PatchGenerator) createEnvironmentVariablePatches(spec *corev1.PodSpec,
 	return patches
 }
 
+func (g *PatchGenerator) removeContainerVolum(volumeMounts []corev1.VolumeMount, pathprefix string, containerId int) k8tz.Patches {
```

Comment:
hi @chenhuazhong, 
first, thank you for finding and fixing the issue!
i did notice some things that we should do before merge:

1. you have used hard-coded `/usr/share/zoneinfo` but its it is a variable, not a constant. please look at `HostPathPrefix` in `PatchGenerator` which hold the correct value (default: `/usr/share/zoneinfo`)
2. we must implement some tests that will reproduce the problematic situation, currently there is no test that able to reach your code (as you can see in the [Codecov report](https://app.codecov.io/gh/k8tz/k8tz/compare/27/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr%20comments&utm_term=k8tz#D1L214)). you can test that the patch is created correctly in `inject_test.go` and you can test that the patch applied and solved the conflict in `transform_test.go`.
3. there is a typo in the `removeContainerVolum` method name (`Volum` => `Volume`)

let's solve this small issues and get this merged!
if you have any questions, i will be more than happy to help :smiley: